po/fr_FR.po

```diff
@@ -16,7 +16,7 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

Comment:
Does this mean that `n != 1` gets plural, so if `n = 0 = plural` ?
Shouldn't it be `n > 1` ?